lib/core/hooks.js

```diff
@@ -6,8 +6,9 @@ import WebpackPlugin from '@sentry/webpack-plugin'
 import { canInitialize, clientSentryEnabled, serverSentryEnabled } from './utils'
 
 const SERVER_CONFIG_FILENAME = 'sentry.server.config.js'
-const SENTRY_DEFAULT_INTEGRATIONS = ['Dedupe', 'ExtraErrorData', 'ReportingObserver', 'RewriteFrames', 'Vue']
+const SENTRY_DEFAULT_INTEGRATIONS = ['CaptureConsole', 'Debug', 'Dedupe', 'ExtraErrorData', 'ReportingObserver', 'RewriteFrames', 'Vue']
```

Comment:
I am not sure about moving `CaptureConsole` & `Debug` to the default integrations (https://docs.sentry.io/platforms/javascript/configuration/integrations/default/). `SENTRY_BROWSER_INTEGRATIONS` var makes more sense to me. 

But I don't know the difference between these two arrays (`SENTRY_BROWSER_INTEGRATIONS` vs `SENTRY_BROWSER_INTEGRATIONS`)

Reply:
The meaning of DEFAULT vs BROWSER is that the DEFAULT maps to `@sentry/integrations` import while BROWSER to `@sentry/browser` import. So it's correct as is.

I admit it's confusing when seen in the context of Sentry documentation but this is just internal naming that users won't see. I can rename DEFAULT to PLUGGABLE I guess.

Reply:
I see. 

> I can rename DEFAULT to PLUGGABLE I guess.

good idea, thx